src/Routing/RouteSubscriber.php

```diff
@@ -69,12 +69,15 @@ protected function alterRoutes(RouteCollection $collection) {
         "entity.entity_view_display.{$entity_type_id}.view_mode" => [
           'bundle' => $entity_type_id,
         ],
+        "layout_builder.defaults.$entity_type_id.view" => [
+          'bundle' => $entity_type_id,
+        ],
       ];
       foreach ($field_ui_routes as $route_name => $defaults) {
-        $route = $collection->get($route_name);
-        assert($route !== NULL);
-        foreach ($defaults as $name => $default) {
-          $route->setDefault($name, $default);
+        if ($route = $collection->get($route_name)) {
+          foreach ($defaults as $name => $default) {
+            $route->setDefault($name, $default);
+          }
```

Comment:
I'm not sure I see any consequential changes here?

Reply:
I see, it's doing null check, which shouldn't truly return null unless the Field UI module is not installed. In that case, we shouldn't run this at all.